### ISSUE TITLE
Add a term for mouse strain ICR/HaJ

### ISSUE THREAD

**Issue by malloryfreeberg** [HumanCellAtlas/ontology#34]
For new term requests, please provide the following information:

## Preferred term label

ICR/HaJ

## Synonyms

ICR

## Textual definition

ICR/HaJ is a mouse strain as described in Jackson Laboratory strain index. This inbred strain is derived from the outbred strain ICR. It may be useful in therapeutic drug testing and genetic analysis of carcinogen-induced colon cancer. [citation](https://www.jax.org/strain/009122)

## Suggested parent term

[Mus musculus (NCBITaxon:10090)](https://ontology.staging.data.humancellatlas.org/ontologies/efo/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FNCBITaxon_10090)

## Use case

Please specify under what circumstances you anticipate this term being used in the HCA metadata.

**Comment by daniwelter** [comment]
http://www.ebi.ac.uk/efo/EFO_0009917
